Invalid pnpm version specified in packageManager field (pnpm@9.8.10 does not exist)

The `package.json` specifies `"packageManager": "pnpm@9.8.10"`, but this version does not exist on the npm registry.

This causes `pnpm install` to fail with:

```
ERR_PNPM_NO_MATCHING_VERSION
```

Suggested fix: change it to `"pnpm@9.8.0"` which is the closest valid version.

Additionally, the `CONTRIBUTING.md` file currently states:

> Make sure you have the matching version of `pnpm` installed by running `pnpm --version`. Currently, we're on `9.8.10`

Correct me if i'm wrong but If `corepack` is used (which is standard with recent Node versions), this step is not necessary , the correct version is automatically resolved based on the `packageManager` field.

--- a/CONTRIBUTING.md
+++ b/CONTRIBUTING.md
@@ -16,11 +16,9 @@ The following steps will get you setup to contribute changes to this repo:
 
 2. Clone your forked repo
 
-3. Make sure you have the matching version of `pnpm` installed by running `pnpm --version`. Currently, we're on `9.8.10`
+3. Run `pnpm install` to install dependencies.
 
-4. Run `pnpm` to install dependencies.
-
-5. Start playing with the code! You can do some simple experimentation in your own project or start implementing a feature right away.
+4. Start playing with the code! You can do some simple experimentation in your own project or start implementing a feature right away.
 
 ### Commands
 

--- a/package.json
+++ b/package.json
@@ -40,7 +40,7 @@
       }
     }
   },
-  "packageManager": "pnpm@9.8.10",
+  "packageManager": "pnpm@9.8.0",
   "dependencies": {
     "@azure/cosmos": "4.2.0",
     "neverthrow": "8.1.1",

